### PR TITLE
Feature#19/styles product service card

### PR DESCRIPTION
**Pull request by AlejandroCuriel**
- What?: We add some styles for the post of products and services. Note: Posts are currently static
 - Why?: We add the static information styles, so that when information is added dynamically it is easier to apply the styles to the cards
  
  ## Screenshots 
Screenshot from 2022-04-26 19-24-35

### CODE REVIEW

File: app/assets/stylesheets/components/_cards.scss [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+  width: 400px;
+
+  h2 {
+    font-weight: 700;
+    margin-top: 1rem;
+    text-transform: capitalize;
+  }
+}
+
+@media screen and (max-width: $mobile) {
```
Comment (xFerchoVJ) [inline app/assets/stylesheets/components/_cards.scss:18]: Nice use of variables!

**Review by JuanequeX — APPROVED** [review]
LGTM

**Review by xFerchoVJ — APPROVED** [review]
LGTM

**Review by Frcomparan — APPROVED** [review]
LGTM

**Pull request merged**